biorbd_optim/optimal_control_program.py

```diff
@@ -497,13 +497,18 @@ def solve(self, solver="ipopt", show_online_optim=False, options_ipopt={}):
         # Solve the problem
         return solver.call(arg)
 
-    def save(self, sol, file_path, to_numpy=False):
+    def save(self, sol, file_path, to_numpy=False, nb_frames=80):
```

Comment:
you can use `**parameters` instead of specific parameters so the user can call any parameters in get_data